platform/ios/sdk-files.json

```diff
@@ -36,7 +36,7 @@
         "platform/ios/vendor/mapbox-events-ios/MapboxMobileEvents/MMETimerManager.m",
         "platform/darwin/src/MGLRasterStyleLayer.mm",
         "platform/darwin/src/MGLForegroundStyleLayer.mm",
-        "platform/ios/vendor/mapbox-events-ios/MapboxMobileEvents/MMENSDateWrapper.m",
+        "platform/ios/vendor/mapbox-events-ios/MapboxMobileEvents/MMEDate.m",
```

Comment:
The nitpitck test is failing, presumably because this PR doesn’t upgrade to the version of the events library that renames this file. Perhaps split this change out into a separate PR?

Reply:
So, I could commit the updated submodule version, but then it would have to be reverted before merging…

Reply:
Went ahead and committed with the events branch selected, will revert before merging